tests/plugins/test_add_filesystem.py

```diff
@@ -293,12 +314,23 @@ def _mockGetTaskResult(task_id):
         }]
     )
 
-    with pytest.raises(PluginFailedException) as exc:
-        results = runner.run()
+    with caplog.atLevel(logging.INFO):
+        with pytest.raises(PluginFailedException) as exc:
+            results = runner.run()
+
     assert task_result in str(exc)
     # Also ensure getTaskResult exception message is wrapped properly
     assert 'image task,' in str(exc)
 
+    if build_cancel:
+        assert "Build was canceled, canceling task 1234567" in caplog.records()[-3].message
```

Comment:
I think its worth to check the order of the messages here, but probably we'd have to change the tests too often for this.

Rewrote this part to use message order only when its necessary - when cancel has thrown an (ignored) error